packages/lambda-tiler/src/routes/tile.ts

```diff
@@ -107,6 +108,56 @@ export async function tileJson(req: LambdaContext): Promise<LambdaHttpResponse>
     return response;
 }
 
+export async function styleJson(req: LambdaContext, fileName: string): Promise<LambdaHttpResponse> {
+    const { version, rest, name } = req.action;
+    const style = fileName.split('.json')[0];
+    const nameComp = TileSetNameParser.parse(name);
+    const host = Env.get(Env.PublicUrlBase) ?? '';
+
+    // Get style Config from db
+    const dbId = Config.Style.id({ name: style }, nameComp.tag);
+    const styleConfig = await Config.Style.get(dbId);
+    if (styleConfig == null) return NotFound;
+
+    // Prepare sources and add linz source
+    const sources: Sources = {};
+    const tileJsonUrl = `${host}/${version}/${name}/${rest[0]}/${rest[1]}/tile.json?api=${req.apiKey}`;
+    for (const [key, value] of Object.entries(styleConfig.sources)) {
```

Comment:
I'm not 100% sold on this logic, I think we need something more complex here. but maybe this will be enough for what we need to do.